presentation-api/controlling-ua/idlharness.html

```diff
@@ -36,15 +36,16 @@
 
 [Constructor(DOMString url)]
```

Comment:
I just check the latest spec at [Interface PresentationRequest](https://w3c.github.io/presentation-api/#idl-def-presentationrequest), the constructor has been changed to `[Constructor(DOMString url),
 Constructor(DOMString[] urls)]`